engine/include/cubos/engine/assets/assets.hpp

```diff
@@ -39,6 +41,10 @@ namespace cubos::engine
     template <typename T>
     using AssetWrite = core::memory::WriteGuard<T, std::unique_lock<std::shared_mutex>>;
 
+    using Type = cubos::core::reflection::Type;
+
+    using cubos::core::reflection::reflect;
```

Comment:
Don't do this in headers - the using statements above are an exception because they're part of the interface of the Assets plugin and they greatly simplify the user's code. These aliases you defined are only useful in the implementation itself.